src/pages/apply/project-manager/ApplyProjectManager.tsx

```diff
@@ -0,0 +1,179 @@
+              </Typography>
+              <Typography variant="body1" gutterBottom className={classes.text}>
+                Project Managers at C4C are responsible for scoping out the features
+                of our projects, prioritizing developer work, and ensuring we're on track
+                to deliver a high quality product at the end of the year. Project Managers
+                lead team and client meetings and with the help of our tech leads adjust and
+                optimize the team's focus and process. They are the main drivers of our initial
+                project specification and help turn that specification into actionable and Agile
+                developer tickets.
+              </Typography>
+            </Box>
+          </Fade>
+          <Fade>
+            <Box mb={5}>
+              <Typography variant="h5" gutterBottom>
+                What we’re looking for
+              </Typography>
+              <Typography variant="body1" gutterBottom className={classes.text}>
+                A great project manager is someone that is very organized and has the foresight
+                to anticipate future challenges. They must be an excellent communicator and have
+                the confidence to lead and manage a team. Experience writing software is a plus,
+                but experience managing any kind of product from start to finish will be more
+                applicable. The ideal candidate is passionate about bringing projects to life
+                and is excited about empowering a team to make an impact in the Boston community.
+              </Typography>
+            </Box>
+          </Fade>
+          <Fade>
+            <Box mb={5}>
+              <Typography variant="h5" gutterBottom>
+                Our ideal candidate is someone who is...
+              </Typography>
+              <Typography variant="body1" className={classes.text}>
+                <ul className={classes.list}>
+                  <li>
+                    Organized and forward thinking
+                  </li>
+                  <li>Passionate about helping the Boston community</li>
+                  <li>
+                    A strong communicator who facilitates collaboration amongst
+                    the product team
+                  </li>
+                  <li>
+                    Interested in being a part of the entire product life cycle:
+                    software inception to completion
+                  </li>
+                  <li>
+                    Dedicated to the C4C mission and has excellent follow
+                    through
+                  </li>
+                </ul>
+              </Typography>
+            </Box>
+          </Fade>
+          <Fade>
+            <Box mb={5}>
+              <Typography variant="body1" gutterBottom className={classes.text}>
+                <i>
+                  All product teams will be working and meeting remotely. <br />
```

Comment:
I know this is just copy-pasted from the other application pages but maybe we want to update this for next semester.

Reply:
Yeah we can edit all the apply pages at once with some PR